### DISCUSSION TITLE
🏖️ Treasure Island

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1956] [Ideas]
The purpose of this discussion is to explore the treasure island proposal, gather feedback and make improvements.

## Treasure Island

Rumour has it that the Bumpkin Pirates that roamed these seas hid their treasure on this very island! Bumpkins are all building boats and heading towards this island in hope that they can dig up and discover hidden gems!

Treasure Island is a new island which provides players the ability to craft shovels and dig in the sand in search for rare items.

<img width="556" alt="Screen Shot 2022-12-30 at 3 56 32 pm" src="https://user-images.githubusercontent.com/11745561/210035977-e1d4b6e3-b50f-4978-a79d-f57589d496f5.png">

**Goal of Treasure Island**

- Provide a sink for resources
- Alternate route for earning SFL, gathering resources and rare items
- An alternate path for players. Expand land or use resources to dig for treasure
- Alternate path to discover food and level your Bumpkin
- A progressive goal for players to reach level 10

### Hidden Treasure

Every 24 hours the tide changes and 7 plots are revealed on treasure island.

By using a treasure tool, you will be able to dig on the plot and find treasure! Once a hole is dug, the tool is burnt. When all 7 holes are dug you must wait for the next day for new holes to appear.

### Treasure Tools

There are 2 tiers of tools available for digging treasure. They can both be used on any plot on the beach.

We estimate an average player to dig between 3-4 holes per day if they dedicate their farming towards it. The additional holes are reserved for players who want to sink resources they have accumulated over time.

**Sand Shovel**

The mighty Sand Shovel is a Bumpkin's favourite tool for finding treasure. 

Wood - 2
Stone - 5
0.0625 SFL
Stock - 10

<img width="171" alt="Screen Shot 2022-12-30 at 3 56 20 pm" src="https://user-images.githubusercontent.com/11745561/210035964-8fcd730e-93dd-4f45-9ca5-9a043e146791.png">

**Metal Detector**

A Metal Detector is an advanced piece of Bumpkin craftsmanship. It enables a Bumpkin to dig more accurately and avoids average treasures.

_For instance, no cactuses or poor bounty_

Iron - 3
Gold - 1
Stock - 5

### Treasure 

The Bumpkin pirates were busy hiding resources, food, rare items + quest items! You can use these items to sell for SFL, level up your Bumpkin or place on your island as rare decorations. https://github.com/sunflower-land/sunflower-land/discussions/1893

Many items will be added to Treasure Island over the weeks, these are the initial available items. At times, some items may be removed from Treasure Island so make sure you start digging early!

**Beach Bounty** - Pirate Bounty, Pearl, Coral, Clam Shell, Pipi, Starfish, Seaweed, Cucumber, Coral, Crab

**Food** - Pirate Cake,  Sunflower Cake, Radish Cake, Carrot Cake, Cauliflower Cake

**Boosts** - Wood Charm (Bonus 0.1 wood), Crop Charm (Bonus 0.05 yield)

**Rare Decorations** - Abandoned Bear, Turtle Bear, Fossil 3, Sunflower Coin, Foliant, Snorkel Bear, Fossil 2, Golden Bear Head, Pirate Bear, Goblin Bear, Galleon, Pirate Hat, Fossil 1, Human Bear, Cactus

**Quest** - Treasure Map

### Selling Beach Bounty

The Goblins at Treasure Island love to collect beach decorations so much that they are willing to buy certain types of treasure from you!

Pirate Bounty - 125 SFL
Pearl - 62.5 SFL
Coral - 25 SFL
Clam Shell - 6.25 SFL
Pipi - 3.125 SFL
Starfish - 1.875 SFL
Crab -0.75 SFL
Cucumber - 0.3125 SFL
Seaweed - 0.125 SFL

### Treasure Map Quest

If you are lucky, you may discover a piece of a treasure map whilst digging for treasure. Gather enough pieces and you are in for a surprise!

### Pirate Bumpkin 

By discovering treasure, you will be able to mint free pieces of Bumpkin Wearables. These will get progressively harder to attain but provide players with added utility when digging for treasures.

_TBD_

**Comment by Polysemouse** [comment]
The metal detector can probably include a small SFL cost to craft like any other tools

**Reply by adamhannigan** [reply]
Yes, makes sense for consistency + SFL burn

**Comment by macon9923** [comment]
Just reading this is fun, :rocket:

**Comment by xSubZero** [comment]
I am completely hyped for this mode, loved all the information!

**Comment by IvanSkazkin** [comment]
I got early access to LE because I'm a top 1 warrior. I bought 45 blue spades for the price, 20 trees and 5 stones. You change the price.
Will there be a refund for blue shovels? You removed them, but many of them were mined at a different price

**Reply by adamhannigan** [reply]
Yes, we will look into this 👍

**Reply by Myth727** [reply]
I bought 65 blue shovels before they got removed btw,  if adjustments I'll be a happy farmer :)

**Reply by Aminbtdds** [reply]
I crafted 40 😄

**Reply by chaoren520** [reply]
Can we return the overpaid materials?

**Comment by Telknub** [comment]
Really liked everything here, can't wait for it. If possible to add the straw hat from Whaitte in #1582 as a bumpkin reward I would love it!

**Comment by Whaitte** [comment]
Can you tell me if Metal Detector already has an art?

**Reply by adamhannigan** [reply]
Not yet, we will be releasing some missing art requirements for this feature shortly.

Feel free to design some ideas for the Metal Detector 🚀

**Comment by Reda2017** [comment]
Hi ! shouldn't the crafting material for sand shovel be swapped ? I mean 5 Wood and 2 Stone instead of the opposite.
the actual situation with tools per sync and the required materiel for tools like pickaxe justify my comment. thanks !

**Reply by adamhannigan** [reply]
We have based these prices on allowing players to craft 3-4 shovels a day from their resources.

We like using more Stone as it adds a Cool Down on resource collection. The other alternative is 20 Wood and 2 Stone.

**Comment by vrillx** [comment]
love it

**Comment by Whaitte** [comment]
I made an art for the metal detector
_V1_
![metal detector .png](https://user-images.githubusercontent.com/104186329/210448905-c21816c4-3c7d-46f7-b5ad-294125d306b5.png)
![metal detector  (1).png](https://user-images.githubusercontent.com/104186329/210448919-45f6007f-fefb-4a83-82a9-32669283403e.png)
_V2_
![metal detector  (2).png](https://user-images.githubusercontent.com/104186329/210448962-af8ca39a-c731-4412-8b30-534f90dd2f3a.png)
![metal detector  (3).png](https://user-images.githubusercontent.com/104186329/210448981-99a868d9-7871-4234-a7c1-cbaa5b1ef447.png)

**Reply by adamhannigan** [reply]
Thanks!

**Comment by NomiceBinocle** [comment]
I find this new game mechanic very exciting, I can't wait to use it

it would just be necessary to change the date on the roadmap because it was planned for January 2 on the whitepaper because it can be confusing for newcomers

**Comment by KUMARGFSTEALER** [comment]
Please increase the Sand Shovel Stock from 10 -> 15 & Metal Detector from 5 -> 7~10 so players who wanted to dig 7x / day will have to sync every 2 days instead of everyday.

Also could you list all the items (average items?) that will be EXCLUDED if we use Metal Detector instead of Sand Shovel?

Providing a list of rarity / percent chance of getting each item for transparency would be great so players won't feel "Scammed" whenever they have bad RNG.

2 Wood + 5 Stone for Sand Shovel doesn't feel right because there's only 30 Pickaxe available and if we were to buy 7 Sand Shovel then we need 35 Pickaxe which is more than 1 sync.

I suggest Sand Shovel should be:
Wood - 10
Stone - 3
0.0625 SFL each
Stock - 15

These numbers are based on the Average Farm of Eligible Farmers (Lvl 10 - 5th Expansion done) which is 7 Trees & 7 Stone.


![image](https://user-images.githubusercontent.com/121823986/210512907-dc4030f1-0c3c-45f6-87f7-7fadae593139.png)

**Reply by adamhannigan** [reply]
On average, we predict players have enough resources to craft 3-4 shovels a day.

For players that want to burn extra resources, we predict they will also craft the Metal Detectors. For these players, they will get a total of 15 tools in a sync - given there are only 7 holes, they can dig for at least 2 days before needing to sync 👍

**Reply by adamhannigan** [reply]
We are still undecided about publicising these numbers.

The core design principle of this feature is surprise & mystery. Players can reverse engineer to figure out the odds, which is fine and this is what a lot of min/max players like to see.

These values aren't secrets, but making it easily known that a Pirate Bounty only drops every (0.1%) will lessen the chance of a player digging for it 😢 

For our casual players, the mystery behind it will be a huge driver to dig again. For players that are into min/maxxing and spreadsheets, they will be able to dig for this data in the community (which is also part of the fun!)

**Reply by KUMARGFSTEALER** [reply]
Agree with both points you mentioned. 

How about the ingredients needed for Sand Shovel?

Current Proposal = 2 Wood + 5 Stone
Alternative (The one you mentioned in one of the comments above) = 20 Wood + 2 Stone
My Suggestion = 10 Wood + 3 Stone

Will you consider this?

**Reply by adamhannigan** [reply]
We are open for any suggestions. Ideally we want to balance this so players can afford 3-4 a day 😄

**Comment by KUMARGFSTEALER** [comment]
Remove Pirate Bear as a Rare Decoration we could get from digging, instead create an Achievement and make Pirate Bear a reward:

Dig 500 Holes in Treasure Island = Pirate Bear

**Reply by adamhannigan** [reply]
That's a good idea - we will want to incorporate some achievements into this. Bears are a great use for achievements

**Reply by Myth727** [reply]
@adamhannigan Any ideas when bears are withdrawable? Great achievements but people like show off their assets on OS and some like to buy/sell. You know the natural crypto user :)

**Reply by adamhannigan** [reply]
This is the next item on our roadmap to pick up ("SFT Issues") 👍 

https://github.com/orgs/sunflower-land/projects/24

**Reply by Myth727** [reply]
Thanks for the response,  did not see that page before. One question,  why focus  on cows before sheep? There is already nft out with boost for sheep, I would assume users wanna start getting utilitie from their farm dog first. Just my 2 cents :)

**Reply by adamhannigan** [reply]
Right now we have more utility for animals that can produce food products (since we have a cooking mechanism).

We will get to sheep, but we need more items that depend on wool as a resource 😄

**Reply by Myth727** [reply]
Tbh honestly I thought clothing minting was gonna require resources like iron for belt bulkes and wool for unique clothing. I figured you guys would make a wool require for many items on bumpkin clothing. But understand your point. Great usecases for wool in real life that can be incorporated in SFL are...


Hügelkultur-look it up, very interesting farming technique.
Clothing
Carpets-like the rugs you had artist design 
Blankets- Blankets for bumkins and animals, we are farming and they do this on real farms. 
Upholstery-Decorations 
Mulch
Fertilizer

I hope this makes you reconsider getting sheep out sooner, figure I just gotta list the usecases and ways we can burn the resources and SFL.

**Comment by Rajat586** [comment]
Can I present my idea.
My idea is if treasure Island is global game server where High quality reward distributed to all game players . Like big land where all players are digging and getting rewards weakly.
Or like fighting with sand monster with sward and those who give high damage they got High reward system in one online server.

**Reply by adamhannigan** [reply]
Nice idea!

The team has recently been discussing these, especially with our MMORPG prototype. It would be awesome to walk around a huge map digging for treasure and claiming land

**Reply by Rajat586** [reply]
Thanks.can i get a chance in early excess

**Comment by Galphath** [comment]
Hi I saw on the proposals that some stuff is missing like the pearl, I hope you like it

**Farm** : 170
**Discord** : La abuela

**PEARL**
![image](https://user-images.githubusercontent.com/20891096/212578400-da2bcd9a-622e-437e-b164-20e92fec0b68.png)

![pearl](https://user-images.githubusercontent.com/20891096/212501577-1bbd105d-9d62-4bc0-8bbd-6a0a65a52314.png)

![image](https://user-images.githubusercontent.com/20891096/212578821-c0087f13-e410-429a-aa9f-7829802c5709.png)

![pearl2](https://user-images.githubusercontent.com/20891096/212578837-4ef34d2b-8710-44ea-9ae8-558883b73f88.png)

Light up

![image](https://user-images.githubusercontent.com/20891096/212589773-cf9dd77b-c3b8-47af-b4c2-8419b91877d3.png)

![pearl alter2](https://user-images.githubusercontent.com/20891096/212589793-62b210a6-82ce-457a-8c03-b0a424a889b3.png)

![image](https://user-images.githubusercontent.com/20891096/212589830-f72f86f9-8125-4347-b21b-c55f611a700a.png)

![pearl alter1](https://user-images.githubusercontent.com/20891096/212589841-03d645a9-495a-4db8-80d1-76f5a544dd74.png)



**PIPI**
![pipi_sample](https://user-images.githubusercontent.com/20891096/212520264-d94a6ce3-a2ab-4f74-a248-a39827a9dde9.png)

1
![pipi1](https://user-images.githubusercontent.com/20891096/212502731-c6678934-a30f-4928-81e3-024e6a3451db.png)
2
![pipi2](https://user-images.githubusercontent.com/20891096/212502732-a387c7a3-dec1-454a-8d90-cb515bea6145.png)

no pearl 
![image](https://user-images.githubusercontent.com/20891096/212577741-99785603-f0c8-440c-9fbc-5948cb23ce1f.png)

3
![pipi3](https://user-images.githubusercontent.com/20891096/212578007-a71b34fa-739f-4e13-9630-a95bd73b5bf6.png)

4
![pipi4](https://user-images.githubusercontent.com/20891096/212578042-f4c33786-a3cc-410b-910f-ec8c2e416ac1.png)


**PIRATE CAKE**
![cake sample](https://user-images.githubusercontent.com/20891096/212520279-b49f6867-7d11-45a2-9c41-44477ce447ef.png)

![pirate_cake](https://user-images.githubusercontent.com/20891096/212520282-48ea3cb8-7ab0-49ca-ab68-c979f88257b5.png)

**SEAWEED**

![image](https://user-images.githubusercontent.com/20891096/212792629-ecdfeb9c-7a6c-4b44-9038-43892eb771c9.png)

![seaweed](https://user-images.githubusercontent.com/20891096/212792717-0da77540-c337-4316-9fae-d68dc9670bec.png)

**Reply by adamhannigan** [reply]
Nice! That Pirate cake is 🔥 👏

**Reply by Galphath** [reply]
Thanks! What do you think about the pearl and pipi? Looking at those again, the pearl could have some variations like how there are black pearls for example and maybe deviate from the game's palette just a tiny bit to make a yellowish one that doesn't clash with the bg,  As for the pipi, maybe a pearl-less version too?

**Comment by digital-farms** [comment]
`land id: 21637`
`discord: KOTOB#4391`

**PEARL**
![pearl_example_v2](https://user-images.githubusercontent.com/105886943/212567772-ef0b5068-94cc-4b54-95df-74d450dd5c27.png)

**Minified:** ![pearl_v2](https://user-images.githubusercontent.com/105886943/212567773-a90f4ffb-a3f4-4752-928d-6ee96d3f5576.png)

V2
![pearl_example](https://user-images.githubusercontent.com/105886943/212536812-d4acb0c4-e441-4701-b798-2dac01db6fcf.png)

**Minified:** ![pearl](https://user-images.githubusercontent.com/105886943/212536813-1050f7d0-0493-479a-8f09-879f95702254.png)

**Comment by mmaxxpeinn** [comment]
Metal Detector
![mdetectr](https://user-images.githubusercontent.com/120295844/212596742-84b19851-4b32-4841-bfaf-f1ded7941f8d.png) ![selectmdetectr](https://user-images.githubusercontent.com/120295844/212596758-77561c5c-50ef-4ac3-acd6-ac590fd796be.png)
minified:
![metaldetector](https://user-images.githubusercontent.com/120295844/212596788-ca91f062-e313-422c-8cef-ff0a2c28eeb3.png)

Pearl and Seaweed
![pearlandseaweed](https://user-images.githubusercontent.com/120295844/212623488-53f98449-e6ee-43cd-bea7-87d3181601a1.png)
minified: 
![pearl](https://user-images.githubusercontent.com/120295844/212623535-0a6114ff-8c36-4bf0-9842-b894511a57cf.png) ![seaweed](https://user-images.githubusercontent.com/120295844/212622483-9f141d0e-05c9-4796-8f1a-87514617e88d.png)

`FarmID: 82122`
`DC: mmaxxpeinn#9351`

**Comment by Galphath** [comment]
**BUMPKINS**  didn't wanted to stretch the other post.

**PENDING** :  

· NPCs


Farm : 170
Discord : La Abuela



**SAMPLES**
![beach samples 400percent](https://user-images.githubusercontent.com/20891096/212814602-1cafaec1-46dc-4470-9bb6-cfd3198e6cbe.png)
![sample1](https://user-images.githubusercontent.com/20891096/212815137-0686ffa3-2b0f-4faa-8abb-e893cc4bc4d3.png)
![Sample2](https://user-images.githubusercontent.com/20891096/212815139-74a0ebda-910c-45cb-8a4c-cf5a52972c2d.png)

**HAT AND HAIRSTYLES**

![Sunhat](https://user-images.githubusercontent.com/20891096/213030785-14156fc9-6687-455a-a754-eb8a8fa50ebf.png)

![SFL2077_hair](https://user-images.githubusercontent.com/20891096/212814088-c2172386-03c6-4f95-a229-82de39d05098.png)
![Sunflare_hair](https://user-images.githubusercontent.com/20891096/212814090-905f2a08-d632-4ad1-9028-cea42bb35df1.png)
![BlackHoleSun_hair](https://user-images.githubusercontent.com/20891096/212815159-4b541884-9bc3-4897-b9f4-8e2a92bb8cfc.png)


**SWIMGEAR**

![Latexswimsuit](https://user-images.githubusercontent.com/20891096/212814276-fae78604-48cc-4583-bc25-997489fa1923.png)
![swimsuit_lines](https://user-images.githubusercontent.com/20891096/212814280-e87b93e4-00ed-4572-878e-c83e1c2c6619.png)
![Bikini](https://user-images.githubusercontent.com/20891096/212814283-3324b832-56bd-42f0-95e1-b5395de4d298.png)

Sarong with texture 

![Sarong_blue](https://user-images.githubusercontent.com/20891096/212814277-a1f5a640-2d45-4dfd-96b8-9e8c3985e433.png)
![Sarong_orange](https://user-images.githubusercontent.com/20891096/212814279-6737666d-0527-48ac-84d3-ca14c25a0db7.png)

Sarong without texture 

![Sarong_orange_noText](https://user-images.githubusercontent.com/20891096/213843648-75e2ec7d-1d66-4881-8949-b053e8981de4.png)
![Sarong_blue_noText](https://user-images.githubusercontent.com/20891096/213843649-ce6abbc2-396b-4a11-b9c2-994508fe4a8f.png)

**SHOES**

![Chancla](https://user-images.githubusercontent.com/20891096/212814325-0215f3a5-19fa-4948-bc53-22379d5fa10c.png)
![sandals](https://user-images.githubusercontent.com/20891096/212814326-e4cf23f6-b0e5-46d3-af4b-f3a3f5ac4980.png)

**HANDHELD**

![Martini](https://user-images.githubusercontent.com/20891096/212814372-44718b93-258b-4c0f-a661-88318f6321f7.png)
![Surfboard](https://user-images.githubusercontent.com/20891096/212814375-ee92420c-ed13-4083-9719-a2d206ae1b8b.png)

**BACKGROUNDS**

![Background](https://user-images.githubusercontent.com/20891096/212814432-9825b623-3fb2-43de-aea4-f77b549b89ef.png)
![BackgroundTowel](https://user-images.githubusercontent.com/20891096/212814434-ca62f8b0-7118-415c-be63-39450a7da46a.png)



-------------------------

**FILES**

[BeachFemaleAssets.zip](https://github.com/sunflower-land/sunflower-land/files/10440026/BeachFemaleAssets.zip)

**Reply by adamhannigan** [reply]
Impressive! @Galphath 

We will definitely be getting some beach bumpkins in there ❤️

**Reply by Whaitte** [reply]
i really need this white hair ❤️❤️

**Comment by zxearl** [comment]
I made this dowsing probe.
If you need it.

![Untitled 11-19-2022 06-50-49](https://user-images.githubusercontent.com/104722435/213907640-fee135e5-9774-4b2f-b137-b1a59c9a850d.gif)
![dowsibg probe](https://user-images.githubusercontent.com/104722435/213907643-78ee5da9-41f5-4022-8228-6e5e9232dfb5.gif)

**Comment by macon9923** [comment]
How about adding Gnome to the Treasure Island like Carrot Sword in Bean !

**Comment by archila79** [comment]
Hello, I am forwarding some work that I do so that you can review it and indicate if it is something useful for the project... thank you


![pixil-frame-0 (15)](https://user-images.githubusercontent.com/115169880/214813674-6506a68b-c86d-4d0d-82b2-1fee8913a6fd.png)
![pixil-frame-0 (5)](https://user-images.githubusercontent.com/115169880/214813785-03f547f6-f495-4f54-9b94-1c499245f077.png)
![pixil-frame-0 (15)](https://user-images.githubusercontent.com/115169880/214813787-557ca2b3-3908-4f74-8952-eca1744812a6.png)
![pixil-frame-0 (14)](https://user-images.githubusercontent.com/115169880/214813789-36f71072-2cb1-4a43-80ce-d3e7b8a97cd3.png)
![pixil-frame-0 (13)](https://user-images.githubusercontent.com/115169880/214813793-4f9b5d03-0c38-4cf5-98a9-f52552151df0.png)
![pixil-frame-0 (12)](https://user-images.githubusercontent.com/115169880/214813794-d99b68a4-a8b4-437b-b754-17c2df4ad7a5.png)
![pixil-frame-0 (11)](https://user-images.githubusercontent.com/115169880/214813796-858512a7-ff25-4254-b71b-b1c09afca42a.png)
![pixil-frame-0 (10)](https://user-images.githubusercontent.com/115169880/214813799-5dbf39d7-5a8c-4c53-8f11-ec69f22c88e6.png)
![pixil-frame-0 (9)](https://user-images.githubusercontent.com/115169880/214813800-69aa818b-e9b8-4dd6-b94d-89b987090ace.png)
![pixil-frame-0 (8)](https://user-images.githubusercontent.com/115169880/214813802-3602b947-f325-4c5e-b7af-5dab7fcbfa1a.png)
![pixil-frame-0 (7)](https://user-images.githubusercontent.com/115169880/214813804-6e426f49-0204-435f-abbe-1bd0345d11c4.png)
![pixil-frame-0 (6)](https://user-images.githubusercontent.co
![pixil-frame-0](https://user-images.githubusercontent.com/115169880/214814009-5bae0bc5-51c7-4ea2-81e0-0d657ca830d8.png)
![pixil-frame-0 (3)](https://user-images.githubusercontent.com/115169880/214814010-8a3b09b9-463c-4459-b21d-6d53f28cc3d0.png)
m/115169880/214813806-873a7901-aa74-450b-b719-a9a2bd055273.png)
